include/libtorrent/enum_net.hpp

```diff
@@ -61,6 +61,7 @@ namespace libtorrent {
 		address interface_address;
 		address netmask;
 		char name[64];
+		// TODO: remove this?
 		int mtu;
```

Comment:
I suppose this was the only thing set from the link. I take it this isn't used anywhere?  believe it was used in the uTP implementation at some point, to initialize the PMTUd. Can you confirm it's not used anymore? (and if so, please remove it)